app/views/home/_social.html.erb

```diff
@@ -1,7 +1,7 @@
-<div id="social">
+<div id="social" style="position: absolute">
```

Comment:
style can be removed

Reply:
I this it will break the functioning of "position:relative" of facebook and twitter buttons.

Reply:
try it once